guide/payments/send.md

```diff
@@ -145,18 +145,18 @@ If your application allows setting spending limits, and the current transaction
 
 {% include image.html
    image = "/assets/images/guide/payments/send/first-confirmation.svg"
-   alt-text = "Notifications after the transaction is broadcasted"
+   alt-text = "Notifications after the transaction is broadcast"
```

Comment:
not sure about this. is it grammatically correct?

Reply:
Both appear to be acceptable. [See here](https://www.merriam-webster.com/words-at-play/is-it-broadcast-or-broadcasted-forecast-or-forecasted-usage). I personally prefer "broadcast", for whatever reason it sounds better, but YMMV.

Reply:
Buffalo buffalo Buffalo buffalo buffalo buffalo Buffalo buffalo